content/docs/act.md

```diff
@@ -0,0 +1,109 @@
+---
+id: act
+title: act
+permalink: docs/act.html
+---
+
+`act()` provides an abstraction to make sure what's rendered reflects a state where pending actions have been 'committed' and the DOM has been updated. When writing UI tests, tasks like rendering, user events, or data fetching can be considered as 'units' of interaction with a user interface. `act` allows you be sure all effects related to a unit have been processed before asserting on the result.
```

Comment:
Can we avoid jargon like "committed", "units", "processing"? This is a bit heavy. There's probably a way to rephrase this with a focus on use cases.